src/sass/components/aside.sass

```diff
@@ -1,20 +1,15 @@
 aside
-  .popular-posts
-    .wpp-category
-      a
-        +is_small-text
-    h4
-      margin-bottom: 15px
+  .tptn_posts
```

Comment:
Perhaps more readable name would be nice here. We generally try to avoid abbreviations.

Reply:
Totally agree! The classes are pre-defined by the plugin already in this case though!